packages/loader/container-loader/src/container.ts

```diff
@@ -869,7 +869,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             assert(!canRetryOnError(error), 0x24f /* "retriable error thrown from attach()" */);
 
             // add resolved URL on error object so that host has the ability to find this document and delete it
-            const newError = DataProcessingError.wrapIfUnrecognized(
+            const newError = CreateProcessingError(
                 error, "errorWhileUploadingBlobsWhileAttaching", undefined);
```

Comment:
this whole flow in weird. why this change at all

Reply:
I followed up with Wes here yesterday, he switched it to `normalizeError`. Didn't look like a data processing error to me.